aten/src/ATen/native/native_functions.yaml

```diff
@@ -11235,3 +11236,11 @@
 - func: unflatten_dense_tensors(Tensor flat, Tensor[] tensors) -> Tensor[]
   variants: function
   python_module: nn
+
+- func: nested_tensor_constructor(Tensor[] list, ScalarType dtype, Device device, bool pin_memory, bool channels_last) -> Tensor
```

Comment:
are the tensor options to handle the case when the input list is empty? Shouldn't these be kwarg only?

Reply:
They can be. This is wrapped within Python, so it doesn't quite matter that much. I can fix this so that we can do away with the Python wrapper and use the native function only.

Reply:
If it's wrapped in Python then give this function a leading underscore.